src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$hover-duration: 300ms;
```

Comment:
it would be better to say that it's exactly the transition duration

Reply:
@ruslanpashkov that is a good recommendation, thanks